react-app/src/constants/chesNotificationStatus.ts

```diff
@@ -18,6 +20,10 @@ export const getStatusString = (status: number): string => {
       return 'Failed';
     case NotificationStatus.Completed:
       return 'Completed';
+    case NotificationStatus.NotFound:
+      return 'Not Found';
+    case NotificationStatus.Invalid:
+      return 'Invalid';
```

Comment:
Same thought. Invalid doesn't seem like a description of the notification status.